easybuild/easyconfigs/p/PETSc/PETSc-3.23.4-foss-2024a.eb

```diff
@@ -0,0 +1,41 @@
+name = 'PETSc'
+version = '3.23.4'
+
+homepage = 'https://www.mcs.anl.gov/petsc'
+description = """PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
+ scalable (parallel) solution of scientific applications modeled by partial differential equations."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+toolchainopts = {'openmp': True, 'usempi': True, 'pic': True}
+
+source_urls = [
+    'https://web.cels.anl.gov/projects/petsc/download/release-snapshots',
+]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['711b2ad46b14f12fe74fcbc7f9b514444646f1e7b20ed57dc7482d34dfc4ca77']
+
+builddependencies = [('CMake', '3.29.3')]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('SciPy-bundle', '2024.05'),
+    ('Boost', '1.85.0'),
+    ('METIS', '5.1.0'),
+    ('SCOTCH', '7.0.6'),
+    ('MUMPS', '5.7.2', '-metis'),
```

Comment:
and the easyblock needs updating for this i believe, it's not building mumps support according to @sassy-crick tests, and indeed i can't see any such indication in the easyblock either.